net/discovery.go

```diff
@@ -120,6 +133,11 @@ func (d *discovery) advertise() {
 			d.provides = []types.ProvidesCoin{types.ProvidesXMR}
```

Comment:
Github won't let me put the comment on line 119, but that line still has the TODO that this PR is closing.

Reply:
removed